test/unit/org/apache/cassandra/cql3/validation/entities/UFSecurityTest.java

```diff
@@ -118,7 +118,13 @@ public void testSecurityPermissions() throws Throwable
                                     "     java.lang.Runtime.getRuntime().exec(\"/tmp/foo\"); return 0d;" +
                                     "} catch (Exception t) {" +
                                     "     throw new RuntimeException(t);" +
-                                    '}'}
+                                    '}'},
```

Comment:
Formatting?

Reply:
What do you want to do with formatting here? I have tried to follow the style of the test. Specifically, what is done for long class names like `org.apache.cassandra.service.StorageService` above.

Reply:
It's all misaligned. The try-catch isn't indented to align with the ones above yes?